target-platform-configuration/src/main/java/org/eclipse/tycho/target/TargetPlatformConfigurationMojo.java

```diff
@@ -1,7 +1,21 @@
+/*******************************************************************************
```

Comment:
@mickaelistria I guessed the copyright header from the git-history, could you confirm that this is correct?

Reply:
Thanks, it's correct.
Shame on me for missing it earlier!